app/src/main/java/com/github/se/signify/ui/screens/profile/ProfileScreen.kt

```diff
@@ -99,14 +110,31 @@ fun ProfileScreen(
             modifier = Modifier.testTag("MyFriendsButton"))
         Spacer(modifier = Modifier.height(32.dp))
 
-        // Statistics Button
-        val myStatsText = stringResource(R.string.my_stats_text)
-        SquareButton(
-            iconId = R.drawable.statisticsicon,
-            text = myStatsText,
-            onClick = { navigationActions.navigateTo(Screen.STATS) },
-            size = 200,
-            modifier = Modifier.testTag("MyStatsButton"))
+        // Number of exercises achieved
+        val exercisesText = stringResource(R.string.completed_exercise_count_text)
+        val easyExercisesText = stringResource(R.string.easy_exercises_text)
+        val mediumExercisesText = stringResource(R.string.medium_exercises_text)
+        val hardExercisesText = stringResource(R.string.hard_exercises_text)
+        StatisticsTable(
+            columnTestTag = "ExercisesColumn",
+            rowTestTag = "ExercisesRow",
+            lineText = exercisesText,
+            statsTexts = listOf(easyExercisesText, mediumExercisesText, hardExercisesText),
+            statsNumberList = listOf("${easy.value}", "${medium.value}", "${hard.value}"),
+            lineTextTestTag = "ExercisesText")
+        Spacer(modifier = Modifier.height(32.dp))
+
+        // Number of quests achieved
+        val questsText = stringResource(R.string.completed_quest_count_text)
+        val dailyQuestsText = stringResource(R.string.daily_quests_text)
+        val weeklyQuestsText = stringResource(R.string.weekly_quests_text)
+        StatisticsTable(
+            columnTestTag = "QuestsColumn",
+            rowTestTag = "QuestsRow",
+            lineText = questsText,
+            statsTexts = listOf(dailyQuestsText, weeklyQuestsText),
+            statsNumberList = listOf("${daily.value}", "${weekly.value}"),
+            lineTextTestTag = "QuestsText")
       },
   )
 }
```

Comment:
## Requested changes

You really should add some padding down here. Your stats are tangent to the navigation bar.

Reply:
I made it in a way that we shouldn't have any screen with their last element glued to the bottom of the screen.